src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
-body {
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+$borderColor: #f3f3f3;
+$fontFamily: "Roboto", sans-serif;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)

Reply:
done